docs/content/en/docs/getting-started/developer/_index.md

```diff
@@ -0,0 +1,215 @@
+    Forwarding from [::1]:8085 -> 8085
+    Forwarding from 127.0.0.1:9042 -> 9042
+    Forwarding from [::1]:9042 -> 9042
+    ```
+
+3. Leave the terminal running in the background.
+
+{{% alert title="Important" color="warning" %}}
+If you close the terminal for any reason, you'll shut down the port forwarding service.
+{{% /alert %}}
+
+## Access K8assandra using the Stargate APIs
+
+[Stargate](https://stargate.io/) provides APIs, data types and access methods that bring new capabilities to existing databases. Currently Stargate adds both REST and GraphQL APIs for CRUD access to data stored in Apache Cassandra® and there are many more APIs coming soon. Separating compute and storage also has benefits for maximizing resource consumption in cloud environments. When using Stargate with Cassandra, you can offload the request coordination overhead from your storage instances onto Stargate instances which has shown latency improvements in preliminary testing.
+
+To access K8ssandra using Stargate:
+
+1. Make sure that all the Stargate services are in order at the Stargate health check endpoint, <http://127.0.0.1:8084/healthcheck>. The pretty-printed raw JSON data should have a value of `true` for each `healthy` key:
+
+   ```json
+    {
+      "bundles": {
+        "healthy": true,
+        "message": "All bundles active",
+        "duration": 0,
+        "timestamp": "2021-02-24T19:06:57.450Z"
+      },
+      "cql": {
+        "healthy": true,
+        "message": "Available",
+        "duration": 0,
+        "timestamp": "2021-02-24T19:06:57.452Z"
+      },
+      "datastore": {
+        "healthy": true,
+        "message": "DataStore is operational",
+        "duration": 3,
+        "timestamp": "2021-02-24T19:06:57.450Z"
+      },
+      "deadlocks": {
+        "healthy": true,
+        "duration": 1,
+        "timestamp": "2021-02-24T19:06:57.452Z"
+      },
+      "graphql": {
+        "healthy": true,
+        "message": "Ready to process requests",
+        "duration": 0,
+        "timestamp": "2021-02-24T19:06:57.450Z"
+      },
+      "restapi": {
+        "healthy": true,
+        "message": "Available",
+        "duration": 0,
+        "timestamp": "2021-02-24T19:06:57.450Z"
+      },
+      "schema-agreement": {
+        "healthy": true,
+        "message": "All schemas agree",
+        "duration": 0,
+        "timestamp": "2021-02-24T19:06:57.450Z"
+      }
+    }
+    ```
+
+2. Generate a Stargate access token replacing `<k8ssandra-username>` and `<k8ssandra-password>` with the values you retrieved in [Retrieve K8ssandra superuser credentials]({{< relref "/docs/getting-started#superuser" >}}):
+
+    ```bash
+    curl -L -X POST 'http://localhost:8081/v1/auth' -H 'Content-Type: application/json' --data-raw '{"username": "<k8ssandra-username>", "password": "<k8ssandra-password>"}'
+    {"authToken":"<access-token>"}
+    ```
+
+3. Use `<access-token>` to populate the `x-cassandra-token` header for all Stargate requests.
```

Comment:
Can't we show how to do one simple request to the REST API here? Seems unsatisfying to get an access token and not do anything with it here.

Reply:
I'm going to recommend we not do that because it's apropos of nothing with respect to getting common tasks completed.